tests/manager/integration/license/opds/opds2/test_api.py

```diff
@@ -51,31 +46,36 @@ def __init__(self, db: DatabaseTransactionFixture, mock_http: MagicMock):
 
         self.api = OPDS2API(db.session, self.collection)
 
+    def queue_default_auth_token_response(self) -> None:
+        """Queue a successful authentication token response."""
+        self.http_client.queue_response(200, content="plaintext-auth-token")
+
     def fulfill(self) -> Fulfillment:
         return self.api.fulfill(self.patron, "", self.pool, self.mechanism)
 
 
 @pytest.fixture
 def opds2_api_fixture(
     db: DatabaseTransactionFixture,
+    http_client: MockHttpClientFixture,
 ) -> Generator[Opds2ApiFixture, None, None]:
-    with patch.object(api, "HTTP") as mock_http:
-        fixture = Opds2ApiFixture(db, mock_http)
-        yield fixture
+    fixture = Opds2ApiFixture(db, http_client)
+    yield fixture
```

Comment:
Minor: We could skip the variable and go straight to the yield here:

```suggestion
    yield Opds2ApiFixture(db, http_client)
```